2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,15 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+ let potentialHeadlines = [];
+    for(let element of allArticleTitles){
+    if(element.length <= 65){
+        potentialHeadlines.push(element);
+    }
+    else continue;
+    }
+ return potentialHeadlines;
+
 }
 
```

Comment:
Look good to me! what is "continue"?  what will happen if you remove else continue?